docs/tutorials/pv_optimization.md

```diff
@@ -0,0 +1,125 @@
+# Optimizing PV production
+
+## Introduction
+
+Before we start it's assumed that you have finished the first [tutorial](./getting_started.md)
+
+In this tutorial you will write an application that optimizes the energy produced from a
+[PV](intro/glossary/#pv) system with Battery for self consumption.
+In order to do so you need to measure the power that flows through the
+[grid connection point](../../intro/glossary/#grid) to determine excess power.
```

Comment:
Once https://github.com/frequenz-floss/frequenz-sdk-python/pull/722 is merged, we'll no longer have to type paths and slugs directly:
```suggestion
{{glossary("Grid", "grid connection point")}} to determine excess power.
```

Reply:
also the dir is not called "intro" anymore, it got renamed to "user-guide".

Reply:
#722 is merged btw, so this is available upstream.